artifactory/utils/golang/project/archive.go

```diff
@@ -3,27 +3,31 @@ package project
 import (
 	"archive/zip"
 	"fmt"
-	"github.com/jfrog/jfrog-cli-go/utils/ioutils"
-	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
-	"github.com/jfrog/jfrog-client-go/utils/log"
 	"io"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strings"
+
+	"github.com/denormal/go-gitignore"
+	"github.com/jfrog/jfrog-cli-go/utils/ioutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
 )
 
 // Archive project files according to the go project standard
 func archiveProject(writer io.Writer, sourcePath, module, version string, excludePathsRegExp *regexp.Regexp) error {
 	zipWriter := zip.NewWriter(writer)
 	defer zipWriter.Close()
 
+	ignore, _ := gitignore.NewFromFile(sourcePath + "/.gitignore")
```

Comment:
The error returned from this function is ignored.

Reply:
Fixed, thanks!.